arm-sql/2015-05-01/swagger/engineAuditRecords.json

```diff
@@ -0,0 +1,225 @@
+                        "in": "path",
+                        "description": "The name of the Azure SQL Database Server on which the Azure SQL Database is hosted.",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "databaseName",
+                        "in": "path",
+                        "description": "The name of the Azure SQL Database for which database engine audit records are retrieved.",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "api-version",
+                        "in": "query",
+                        "description": "Client Api Version.",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "$filter",
+                        "in": "query",
+                        "description": "The filter to apply on the operation",
+                        "required": false,
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Successfully retrieved the list of engine audit records.",
+                        "schema": {
+                            "$ref": "#/definitions/EngineAuditRecordResourceList"
+                        }
+                    },
+                    "default": {
+                        "description": "*** Error Responses: ***\n\n * 400 InvalidEngineAuditRecordsQueryParameters - Invalid query parameters\n\n * 400 InvalidEngineAuditRecordsShowSystemLogsFilter - The ShowSystemLogs property only supports the equal (eq) operation\n\n * 400 InvalidEngineAuditRecordsEndTimeFilter - The EventTime property currently supports only the less than or equal (le) operation, designating the End Time.\n\n * 400 InvalidEngineAuditRecordsShowServerRecordsFilter - The ShowServerRecords property only supports the equal (eq) operation\n\n * 404 SourceDatabaseNotFound - The source database does not exist.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 405 DatabaseEngineAuditRecordsNotImplemented - Operations on database audit records are not enabled yet\n\n * 500 GetEngineAuditRecordsFailed - Failed to get audit records",
+                        "schema": {
+                            "$ref": "#/definitions/ArmErrorResponseMessage"
+                        }
+                    }
+                },
+                "deprecated": false,
+                "x-ms-odata": "#/definitions/EngineAuditRecordsFilterParameters",
+                "x-ms-pageable": {
+                    "nextLinkName": "nextLink"
+                }
+            }
+        }
+    },
+    "definitions": {
+        "Object": {
```

Comment:
removed

---
In reply to: [80169355](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80169355) [](ancestors = 80169355)